src/state/content-settings/actions.js

```diff
@@ -82,10 +84,18 @@ export const sendPostReloadReferences = () => dispatch => new Promise((resolve)
 export const sendPostReloadIndexes = () => dispatch => new Promise((resolve) => {
   dispatch(toggleLoading('reloadIndexes'));
   postReloadIndexes().then((response) => {
-    response.json().then((json) => {
+    response.json().then(async (json) => {
```

Comment:
can you please use only async/await in this function instead of mixing it with promises?